data_pipeline_api/file_formats/parameter_file.py

```diff
@@ -5,6 +5,7 @@
 import toml
 import numpy as np
 from scipy import stats
+from math import sqrt
```

Comment:
numpy also has a square root (`np.sqrt`) and python also allows non-integer powers (i.e. `x ** 0.5`), but I don't see any reason to change it from what it is.